doc/tutorial/00_first_example/index.md

```diff
@@ -0,0 +1,28 @@
+# A first example {#tutorial_first_example}
+
+[TOC]
+
+This tutorial walks you through a small SeqAn3 program. It is intended to give you a short overview of what to expect
+in the other tutorials and how to use this documentation.
+
+<table>
+<tr>  <th> Difficulty             </th>  <td> Easy                   </td>   </tr>
```

Comment:
can you copy and paste an example of how this looks like in doxygen?

Reply:
![screenshot-2019-02-10-15-50-23](https://user-images.githubusercontent.com/2996863/52535209-804b7b80-2d43-11e9-81e4-09d8133af0b8.png)

Reply:
Thanks :) looks alright. Do we still plan to port the style to "Read The Docs" at some point?